examples/applications/plot_stock_market.py

```diff
@@ -74,14 +25,15 @@
 
 from sklearn import cluster, covariance, manifold
```

Comment:
could you move each of the imports above in the cell where they are used the first time?

Reply:
Yeah.